### ISSUE TITLE
Have you benchmarked this?

### ISSUE THREAD

**Issue by ethancaballero** [onlytailei/A3C-PyTorch#1]
How well does it do on Breakout, etc.?

**Comment by onlytailei** [comment]
I'm still testing it. There will be a result soon.

**Comment by onlytailei** [comment]
https://www.youtube.com/watch?v=Ohpo6BcMgZw&feature=youtu.be 
please check it.

**Comment by ethancaballero** [comment]
Train it on Breakout for a day. Pong's too easy.

**Comment by andrewliao11** [comment]
one quick question: what's the different btn this repo and [ikostrikov's](https://github.com/ikostrikov/pytorch-a3c)

**Comment by onlytailei** [comment]
Have to say that now it is very similar.
At the beginning, I want to calculate the loss of former steps as a batch. But somehow, it does not work.
